python/docs/source/getting_started/install.rst

```diff
@@ -68,8 +68,8 @@ It is recommended to use ``-v`` option in ``pip`` to track the installation and
 
     HADOOP_VERSION=2.7 pip install pyspark -v
 
-Supported versions of Hadoop are ``HADOOP_VERSION=2.7`` and ``HADOOP_VERSION=3.2`` (default).
-Note that this installation of PySpark with a different version of Hadoop is experimental. It can change or be removed between minor releases.
+Supported values in ``HADOOP_VERSION`` are ``without``, ``2.7`` and ``3.2`` (default).
```

Comment:
I might want to know what `without` means, like it's equivalent to `user-provided` one or something.

Reply:
Sure, let me update.